README.md

```diff
@@ -1,40 +1,146 @@
-# Introduction to Python
-Before starting working on this project
+# Introduction to Python - Please read!
```

Comment:
maybe adding a little smiley at the end of the sentence will make it a little warmer welcome :):)